subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/OutputScrapingExecutionResult.java

```diff
@@ -167,14 +167,18 @@ private String normalize(LogContent output) {
                 result.add(BUILD_RESULT_PATTERN.matcher(line).replaceFirst("$1 $2 in 0s"));
                 i++;
             } else {
-                result.add(line);
+                result.add(normalizeLambdaIds(line));
                 i++;
             }
         }
 
         return LogContent.of(result).withNormalizedEol();
     }
 
+    private String normalizeLambdaIds(String line) {
```

Comment:
Add some Javadoc explaining why we need this.

Reply:
Done.